lib/access_check_filter.js

```diff
@@ -64,6 +66,9 @@ module.exports = (hyper, req, next, options, specInfo) => {
                 }
 
                 return contentPromise.then((content) => redirect(content, location, options));
+            } else if (content.status === 302) {
```

Comment:
To me it looks as if this condition would fire when content *was* in storage, but the restriction table did not have the redirect. Is that intended?